ansible/roles/analytics-bootstrap-always/tasks/main.yml

```diff
@@ -26,7 +27,7 @@
   with_items:
     - rsync
     - zip
-    - python-pip
+    - python3-pip
```

Comment:
@santhosh-tg Can we check `ansible_distribution_version` var before installing this? Please follow similar logic as this file `ansible/roles/provision-geo-location-db/tasks/main.yml`